src/wasi/Wasi.cpp

```diff
@@ -16,8 +16,12 @@
 
 #include "wasi/Wasi.h"
 
+// https://github.com/WebAssembly/WASI/blob/main/legacy/preview1/docs.md
+
 namespace Walrus {
 
+std::string WASI::m_wasiErrorMessage[WASI::wasi_errno::notcapable + 1];
```

Comment:
It seems that this string list is not used at all.
Would it be used later?

Reply:
I would like to use it to somehow communicate errors to the users of these functions, but I was unsure how you would like this part of wasi, so for the time being I left it it. It contains all the error messages in pairs with the error codes.

Reply:
`walrus` aims to be a lightweight WASM engine for resource-limited devices.
So, allocating a global array just for error messages does not fit in our goal.
If you really need this array, please use it with debug mode only.